docs/v3/tutorials/server-and-worker.mdx

```diff
@@ -1,9 +1,9 @@
 ---
-title: Deploy a Prefect Server and Worker with Helm
-description: Deploy a Prefect OSS Server and Worker with Helm.
+title: Deploy a Self Hosted Prefect Server and Prefect Worker with Helm
```

Comment:
we use the term `self hosted` else where in our docs and in the helm repo -- adjusting here for ocnsistency